ch_pipeline/core/io.py

```diff
@@ -35,6 +35,7 @@
 
 from caput import pipeline
 from caput import config
+from caput import mpiutil
```

Comment:
I'm not quite sure why you need to add this import?

Reply:
Right, that import is not needed for this PR.

My editor flagged a few lines in this file because `mpiutil` is missing, so it seems that perhaps the import _is_ needed, but that probably shouldn't be part of this PR.

I removed it.